variables-and-functions.org

```diff
@@ -0,0 +1,124 @@
+** Variables
```

Comment:
I went over dashboard.el and dashbord-widgets.el and noted the functions/variables and their arguments/uses.
Without this I, as a pretty new user, have had to go to the src code to checkout all variables and functions.
I see a lot of other projects having similar tables in their READMEs, but didn't want to invade so much of the first-page real estate since you all didn't build it.

As for maintenance, apart from my examples which might be too personalised, unless you add or plan on adding much functions and variables, it should be straightforward to add a line to either table.